source/shared/net/desktop-wallet/shield-gtu-desktop.rst

```diff
@@ -13,7 +13,10 @@ Shield GTU on an account
 Shield GTU
 ==========
 
-When you shield an amount on an account, only the account's credential holder can see the shielded amounts. Other participants in the network will be able to see the shielding transaction, but can't see the shielded balance or any shielded transfers going in or out of the account. You can only make shielded transfers on accounts with a single credential, and if you have an account with GTU in the shielded balance, you can't add new credentials to that account.
+When you shield an amount on an account, only the account's credential holder can see the shielded amounts. Other participants in the network will be able to see the shielding transaction, but can't see the shielded balance or any shielded transfers going in or out of the account.
+
+.. Note::
+   You can't make shielded transfers on multi-signature accounts, only on accounts with a single credential. Likewise, if you have an account with GTU in the shielded balance, you can't add new credentials to that account.
```

Comment:
Can people add more credentials to accounts that now have 0 GTU on the shielded balance, but which previously had some GTU on the shielded balance? Meaning there are transactions in the transfers log.